VideoToImage/.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
you should just have to add one gitignore in the root for the files you want to globally ignore.